source/Grid/Grid.jest.js

```diff
@@ -1951,6 +1952,48 @@ describe('Grid', () => {
       expect(cellRendererCalls.length).toEqual(0);
     });
 
+    it('should not trigger render by _debounceScrollEndedCallback if process slow table', async () => {
+      const scrollingResetTimeInterval = 50;
+      const cellRangeRendererCalls = [];
+      function cellRangeRenderer(props) {
+        const startTime = Date.now();
+        while (Date.now() - startTime <= scrollingResetTimeInterval); // imitate very slow render
+        cellRangeRendererCalls.push(props);
+        return defaultCellRangeRenderer(props);
+      }
+      const props = {
```

Comment:
I believe the only prop you need to override from the default is `scrollingResetTimeInterval` 

`getMarkup` https://github.com/bvaughn/react-virtualized/blob/master/source/Grid/Grid.jest.js#L44 provides the required default props which shouldn't additionally specified unless necessary for the test.